alttprbot_racetime/bot.py

```diff
@@ -20,6 +20,10 @@ def start_racetime(loop):
 
 racetime_bots = {}
 for slug, category in RACETIME_CATEGORIES.items():
+    if not category.client_id:
+        raise ValueError("Racetime category %s has no client id" % slug)
+    if not category.client_secret:
+        raise ValueError("Racetime category %s has no client secret" % slug)
```

Comment:
When running python sahasrahbot.py locally without any environment variables set, it would fail on an HTTPError.  I thought it'd be better if it failed because the variables are not set.